rasa/core/training/interactive.py

```diff
@@ -424,6 +416,7 @@ def _selection_choices_from_intent_prediction(
         choices, sender_id, endpoint
     )
 
+    # TODO: Should we not always set the confidence = 1.0?
```

Comment:
Which scenarios do you mean? The scenario in which the user selected a specific intent? Maybe, something to discuss in Slack?

Reply:
Discussed it with Akela, and we now set it to 1.0